app/src/main/java/com/example/thriftbooks/fragments/HomeFragment.java

```diff
@@ -87,7 +87,7 @@ public void onRefresh() {
         scrollListener = new EndlessRecyclerViewScrollListener(linearLayoutManager) {
             @Override
             public void onLoadMore(int page, int totalItemsCount, RecyclerView view) {
-                queryPosts(allPosts.size());
+                queryPosts(0);
```

Comment:
Why is this 0?

Reply:
I wanted to get the recent posts while in EndlessScrolling 
But I can make it back to allposts.size()